docs/api/cli-options.md

```diff
@@ -74,8 +80,127 @@ Options include:
 | `--disable-telemetry`           | Disables Storybook's telemetry. Learn more about it [here](../configure/telemetry.md).<br/>`storybook build --disable-telemetry`                                                                      |
 | `--test`                        | Optimize Storybook's production build for performance and tests by removing unnecessary features with the `test` option. Learn more [here](../api/main-config-build.md).<br/>`storybook build --test` |
 
-<Callout variant="info" icon="💡">
+### `add`
```

Comment:
If we're not going to put these in alphabetical order, then I would think `upgrade` would be used more often than `add`. I'd also place `remove` immediately after `add`. So the final order (after `build`) would be:

1. `upgrade`
2. `add`
3. `remove`
4. `doctor`
5. `info`
6. `sandbox`

Reply:
Also, just realized `init` is missing. I'd put that first, above `upgrade`.

Reply:
Based on this, I'm more inclined to:
- init
- add
- remove
- upgrade
- doctor
- info
- sandbox

Init to set up Storybook, which is followed by the `add`/ `remove` commands that is the natural option as they'll be adding and removing addons constantly, upgrade and doctor, which are self-explanatory and the last two for issues bugs. Are you good on this, @shilman ? Sounds reasonable to you?

Reply:
I think @jonniebigodes's suggestion is good too, especially if we want users to use `add` more than they do today.

Reply:
That's what I was aiming for @JReinhold , to have the `add` command become more relevant and become the default for installing addons.